test/runtests.jl

```diff
@@ -44,61 +45,70 @@ setprogress!(false)
 
 include(pkgdir(Turing)*"/test/test_utils/AllUtils.jl")
 
+# Collect timing and allocations information to show in a clear way.
+const TIMEROUTPUT = TimerOutputs.TimerOutput()
+macro timeit_include(path::AbstractString)
+    return :(@timeit TIMEROUTPUT $path include($path))
+end
```

Comment:
I guess it even fits in one line?

```suggestion
macro timeit_include(path::AbstractString) = :(@timeit TIMEROUTPUT $path include($path))
```

Reply:
That's not valid syntax. It would be possible to do
```suggestion
macro timeit_include(path::AbstractString) :(@timeit TIMEROUTPUT $path include($path)) end
```
but the 3-line version seemed more readable.

Reply:
Ah well it's not too bad